assets/app/scripts/app.js

```diff
@@ -160,6 +160,26 @@ angular
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
   .constant("AUTH_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).auth))
+  .factory("ProxyPod", function(API_CFG) {
+    return function(namespace, podId, port) {
+      var kubeProxyURI = new URI("https://" + API_CFG.k8s.hostPort + API_CFG.k8s.prefix);
```

Comment:
data.js uses the current location's scheme, rather than hard-coding `https`... not sure if we want to do the same here